conformity/resource_conformity_custom_rule.go

```diff
@@ -370,6 +370,8 @@ func processInputCustomRuleConditions(conditionsIn []interface{}) []cloudconform
 			obj.Value, _ = strconv.ParseBool(m["value"].(string))
```

Comment:
⛏️ Could you also update comment block up to to include `null` (where it says string, boolean, integer...)